packages/cli/src/events/relay-event-map.ts

```diff
@@ -11,12 +11,13 @@ export type UserLike = {
 	role: string;
 };
 
-/**
- * Events sent by `EventService` and forwarded by relays, e.g. `AuditEventRelay` and `TelemetryEventRelay`.
- */
-export type Event = {
+export type RelayEventMap = {
 	'server-started': {};
 
+	/**
+	 * Workflow
+	 */
```

Comment:
Should we use [VSCode foldable regions](https://code.visualstudio.com/docs/editor/codebasics#_folding) for these? E.g.

```ts
//
// #region Workflow
//

...

// #endregion
```

or

```ts
// #region Workflow

...

// #endregion
```

That way they can be folded. This is of course VSCode only feature, but personally I like it as a way to organize regions of code in files.